### ISSUE TITLE
Killed

### ISSUE THREAD

**Issue by Ai-Artsca** [XavierXiao/Dreambooth-Stable-Diffusion#33]
keep getting this error that its been Killed lol

/venv/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py:2102: LightningDeprecationWarning: Trainer.root_gpu is deprecated in v1.6 and will be removed in v1.8. Please use Trainer.strategy.root_device.index instead.
rank_zero_deprecation(
Epoch 0: 0%| | 0/1010 [00:00<?, ?it/s]/venv/lib/python3.8/site-packages/pytorch_lightning/utilities/data.py:72: UserWarning: Trying to infer the batch_size from an ambiguous collection. The batch size we found is 1. To avoid any miscalculations, use self.log(..., batch_size=batch_size).
warning_cache.warn(
/venv/lib/python3.8/site-packages/pytorch_lightning/trainer/connectors/logger_connector/result.py:229: UserWarning: You called self.log('global_step', ...) in your training_step but the value needs to be floating point. Converting it to torch.float32.
warning_cache.warn(
Epoch 0: 0%| | 1/1010 [00:02<47:26, 2.82s/it, loss=0.0249, v_num=0, train/losHere comes the checkpoint...
Killed

**Comment by SpaceADV** [comment]
same here exactly the same

**Comment by SpaceADV** [comment]
"If you get the error about it being killed after 1 step, open the terminal, type "ps aux" and look for the pid for both python relauncher and webui, then type "kill (the id for either)" and kill both of them.  Was stuck on that error for a while with an A5000 but this fixed my problem." from a youtube comment by Wanky, this worked for me

**Comment by WingTangWong** [comment]
> "If you get the error about it being killed after 1 step, open the terminal, type "ps aux" and look for the pid for both python relauncher and webui, then type "kill (the id for either)" and kill both of them. Was stuck on that error for a while with an A5000 but this fixed my problem." from a youtube comment by Wanky, this worked for me

omg. Thanks for this!!

**Comment by obkorolev** [comment]
Worked for me!! Thanks a lot!
